config/config_cheyenne_gnu-10.sh

```diff
@@ -30,9 +31,14 @@ module load ncarenv/1.3
 # Load these basic modules for Cheyenne
 module load cmake/3.18.2
 
+# Load Python environment
+module use /glade/p/ral/jntp/tools/miniconda3/4.8.3/modulefiles
+module load miniconda3/4.8.3
```

Comment:
This module should not be loaded in config; just like we don't load the compiler in the config.
If for e.g. netcdf is built in miniconda, it can cause all kinds of headaches when building the software.

Reply:
Ok, this explanation makes sense to me. Note that the config files for Cheyenne and Gaea do (and always did) contain module load/switch statements for the compiler (and MPI). I don't think it works on Cheyenne otherwise because of the way CISL designed the ncarcompilers and ncarenv modules.

Reply:
I was able to build everything with the changes I just pushed for the cheyenne and gaea config*.sh files. I didn't get the hpc python modules, presumably because I did not create the miniconda meta modules. I really don't know what I would be doing there, can I ask you to push a miniconda module to my branch?